src/PlaneWaveBasis.jl

```diff
@@ -359,11 +359,9 @@ G_vectors(::PlaneWaveBasis, kpt::Kpoint) = kpt.G_vectors
 
 The list of ``G`` vectors of a given `basis` or `kpt`, in cartesian coordinates.
 """
-function G_vectors_cart(basis::PlaneWaveBasis)
-    map(G -> basis.model.recip_lattice * G, G_vectors(basis))
-end
+G_vectors_cart(basis::PlaneWaveBasis) = recip_vector_red_to_cart.(basis.model, G_vectors(basis))
 function G_vectors_cart(basis::PlaneWaveBasis, kpt::Kpoint)
-    map(G -> basis.model.recip_lattice * G, G_vectors(basis, kpt))
+    recip_vector_red_to_cart.(basis.model, G_vectors(basis, kpt))
 end
```

Comment:
Let's hope this won't become too much of a hassle with the autodiff.

Reply:
no particular reason why it should: it's exactly equivalent to the previous version